src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/AutoReloading.kt

```diff
@@ -0,0 +1,37 @@
+package nl.avisi.structurizr.site.generatr.site.views
+
+import kotlinx.html.*
+import nl.avisi.structurizr.site.generatr.site.asUrlRelativeTo
+import nl.avisi.structurizr.site.generatr.site.model.PageViewModel
+
+fun HEAD.script(viewModel: PageViewModel) {
```

Comment:
With this name, it's hard to distinguish between the HTML `script` tag and this function.

```suggestion
fun HEAD.autoReloadScript(viewModel: PageViewModel) {
```